_posts/2021-05-22-fluxv2-openfaas.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "See how Flux's Git Ops ToolKit can simplify delivery"
+description: "See how Flux can simplify serverless deployments to OpenFaaS"
+date: 2021-05-22    
+image: /images/2021-05-22-flux/earth.jpg
+categories:
+- arkade
+- kubectl
+- gitops
+- openfaas-operator
+- flux
+- CI/CD
+author_staff_member: alistair
+dark_background: true
+
+---
+
+We learn how to manage OpenFaaS and install functions using Flux.
+
+# What is Flux
+Flux is an open source project by [weaveworks](https://www.weave.works/oss/flux/) which enables 
+application delivery using GitOps. Your cluster config is stored in Git and the flux controllers continually work to ensure
+that the cluster config matches the state in Git. This allows us to take the "cattle not pets" mentality applied to modern cloud computing further and apply the same thinking
+to our clusters. 
+
+We no longer need to manually issue `helm install` or `kubectl apply` commands to manage our clusters and 
+configuration changes are audited and easily reviewed as Git PullRequests.
+
+![flux v2 diagram](/images/2021-05-22-flux/gitops-toolkit.png)
+*image from [Flux](https://github.com/fluxcd/flux2/blob/main/docs/_files/gitops-toolkit.png)*
+ 
+# GitOps
+
+`GitOps` is a set of practices that aim to democratise our "Operations" type practices such as deploying new application 
```

Comment:
Let's remove this section because we have had several blog posts recently all of which redefine GitOps. Link to your favourite definition instead. Plus you already talked about it in the previous section.

https://www.gitops.tech
https://www.weave.works/technologies/gitops/